frontend/src/pages/BookGalleryPage/components/searchbar/Searchbar.tsx

```diff
@@ -0,0 +1,18 @@
+import './Searchbar.css'
+
+export default function Searchbar({setSearchInput}: any) {
+    return (
+        <>
+            <label className={"search-label"} htmlFor={"searchbar"}>
```

Comment:
so far in the project, the labels don't go around the input (that's why we have the htmlFor), instead I would just a div for styling that includes the label and the input